diff -r 19aa387ec1a0 bin/contrib/webserver.rc
--- a/bin/contrib/webserver.rc	Tue Oct 20 02:28:10 2009 +0200
+++ b/bin/contrib/webserver.rc	Thu Apr 22 16:40:06 2010 +0100
@@ -20,11 +20,11 @@
 	file='404.html'
 }
 
-echo 'HTTP/1.1 ' ^$response
-echo 'Date: ' `{date}
-echo 'Server: rc shell'
-echo 'Content-Length: ' `{cat $file | wc -c | tr -d ' '}
-echo 'Content-Type: ' `{file -i $file  | awk '{ print $2 }'}
-echo 'Connection: close'
+echo    'HTTP/1.1 ' ^$response
+echo -n 'Date: ' `{date}
+echo    'Server: rc shell'
+echo -n 'Content-Length: ' `{cat $file | wc -c | tr -d ' '}
+echo -n 'Content-Type: ' `{file -i $file  | awk '{ print $2 }'}
+echo    'Connection: close'
 echo
 cat $file
